compiler/src/dmd/func.d

```diff
@@ -2197,6 +2197,21 @@ extern (C++) class FuncDeclaration : Declaration
         }
         else
         {
+            foreach (v; closureVars)
+            {
+                doNotInferScope(v, null);
```

Comment:
This call is redundant with `notMaybeScope`

Reply:
fixed